manage.py

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python 
```

Comment:
I think it is accidental.

Reply:
I added to re-run the circle ci build. Removed now